salt/modules/win_update.py

```diff
@@ -83,6 +85,11 @@ def __virtual__():
     Only works on Windows systems
     '''
     if salt.utils.is_windows() and HAS_DEPENDENCIES:
+        salt.utils.warn_until(
+            'Oxygen',
```

Comment:
This needs to be one more version farther out to give people a chance to switch. Our policy is 2 feature releases out, so can you change the `Oxygen` versions to `Fluorine`? [Doc Reference](https://docs.saltstack.com/en/latest/topics/development/deprecations.html#deprecating-code)